seirmo/model.py

```diff
@@ -0,0 +1,37 @@
+#
+# ForwardModel Class
+#
+
+
+class ForwardModel:
+    """ForwardModel Class:
+
+    Abstract base class for the SEIR model class.
+
+    Parameters
+    ----------
+    value: numeric, optional
+        example of value
+
+    """
+
+    def __init__(self):
+
+    def simulate(self):
+        """
+        Forward simulate the daily number of infections for a given time period with given parameters
+
+        Parameters
+        ----------
+        parameters: sequence of numerics
+        times: sequence of numerics
+
+        Returns
+        ----------
+        n_outputs: numpy array of 1- or 2-dimmension
```

Comment:
dimmension -> dimension

Reply:
The output will probably just called `output`. `n_output` sounds like it is referring to the number of outputs.